app/controllers/api/users_controller.rb

```diff
@@ -3,7 +3,8 @@ class UsersController < Api::ApplicationController
     skip_before_action :doorkeeper_authorize!, only: %i[create]
 
     def create
-      user = User.new(name: user_params[:name], email: user_params[:email], password: user_params[:password])
+      user = User.new(name: user_params[:name], email: user_params[:email], password: user_params[:password],
```

Comment:
_note: Since there is no open PR in the frontend project for me to review, I'll include my comments here._

# Frontend review

- [ ] On your frontend readme, there are no instructions about how to setup and run the app:

![image](https://user-images.githubusercontent.com/57421823/233584190-9248982f-1eb7-46a3-982a-5feb969b7cd3.png)

Kindly, complete those instructions so your users are able to run the app smoothly. 👍 

- [ ] The "Register" button is not working for me:

![image](https://user-images.githubusercontent.com/57421823/233586973-a9afafaf-ba06-4361-aafa-5b283406151e.png)

No matter how many times I click on it, nothing happens.  

Kindly fix this button so new users can be created, and your app works as intended.

_This means I cannot create new users using the frontend. Continuing the review with the admin user found in `seeds.rb`_

_Tested on both Chrome and Firefox._

- [ ] As an Admin user, I couldn't find a way to add an apartment in the frontend.  This is a core feature requirement.

If this feature already exists on your project, kindly make it more obvious.  If it does not exist, kindly implement it so this particular requirement is met.

![image](https://user-images.githubusercontent.com/57421823/233590556-09ee9728-c83c-4ea0-b988-c56accf958c3.png)


_note: If for some reason, you weren't able to implement it, kindly add a proper explanation on your Pull request description._